appsdk/sdk.go

```diff
@@ -305,7 +305,7 @@ func (sdk *AppFunctionsSDK) Initialize() error {
 	bootstrapComplete := false
 
 	// Bootstrap retry loop to ensure all dependencies are ready before continuing.
-	until := time.Now().Add(time.Millisecond * time.Duration(internal.BootTimeoutDefault))
+	until := time.Now().Add(internal.BootTimeoutDefault)
```

Comment:
How do we use `Service.BootTime` from configuration, rather than just `internal.BootTimeoutDefault`

Reply:
I do not think you can... only after configuration been loaded into memory then you can but here is just trying to that like chicken and egg problem here

Reply:
Yea, then  `Service.BootTime` from configuration is never used.

Reply:
Unless we want to restructure this `for` loop so that the `Service.BootTime` starts after `sdk,initializeConfiguration()` in line 311 is done.  I guess we could do that way...just a bit complicated...